### DISCUSSION TITLE
希望直播间界面增加夜间模式开关

### DISCUSSION THREAD

**Discussion by Ranlajetech** [the1812/Bilibili-Evolved#1502] [功能建议 / Ideas]
如题，在当前版本（Stable v1.11.12 内容包: ddfe6db）下，想要手动更改直播间界面明暗模式，需要先跳转到其他页面（例如主站）进行切换再刷新直播页面，希望作者能在直播页面下添加这个开关，十分感谢。

**Comment by the1812** [comment]
用不着刷新啊, 左边不是有设置吗
